lodash.defaultsdeep/index.js

```diff
@@ -1146,8 +1166,8 @@ function baseMerge(object, source, srcIndex, customizer, stack) {
  *  counterparts.
  */
 function baseMergeDeep(object, source, key, srcIndex, mergeFunc, customizer, stack) {
-  var objValue = object[key],
-      srcValue = source[key],
+  var objValue = safeGet(object, key),
```

Comment:
This is backport from new version: https://github.com/snyk/lodash/blob/4.17.12-pre/lodash.js#L3624